plugins/Ntuplizer.cc

```diff
@@ -458,7 +458,7 @@ Ntuplizer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
     //------------------------------------------------------------------------------------------
     //------------------------------------------------------------------------------------------
     if(isVerbose) std::cout << "HT" << std::endl;
-    HT = theCHSJetAnalyzer->CalculateHT(iEvent,iSetup,3,15,3.);
+    HT = theCHSJetAnalyzer->CalculateHT(iEvent,iSetup,2,15,3.,true);
```

Comment:
Nice to have a separate flag as far as the JetAnalyzer is concerned, but the value is still hardcoded in the Ntuplizer itself. @lbenato, would it be useful have a dedicated switch e.g. `SmearHTJets` in the config file(s)? 

In fact, anticipating additional changes from the upcoming PR's, I'm wondering if we should have a dedicated `HTSet`. This could include e.g. the configurable `minHT` event preselection that is already used for the tracking phase space (and will be pushed to the repo once my turn comes). The kinematics/ID cuts on the jets that go into HT calculation (also hardcoded in this line here) could also be made configurable and included in such PSet. If you would find that useful, I can implement it together with the other changes I'll be pushing.

Reply:
Yes indeeed it would be a good feature to implement, thanks!